seqr/views/apis/dataset_api_2_3_tests.py

```diff
@@ -190,12 +197,12 @@ def test_add_variants_dataset(self, mock_es_search, mock_get_index_metadata, moc
         self.assertSetEqual(set(response_json.keys()), {'samplesByGuid', 'individualsByGuid', 'familiesByGuid'})
         sv_sample_guid = 'S1234567_NA19675_1'
         self.assertDictEqual(response_json['familiesByGuid'], {})
-        self.assertListEqual(response_json['samplesByGuid'].keys(), [sv_sample_guid])
+        self.assertListEqual(list(response_json['samplesByGuid'].keys()), [sv_sample_guid])
         self.assertEqual(response_json['samplesByGuid'][sv_sample_guid]['datasetType'], 'SV')
         self.assertTrue(response_json['samplesByGuid'][sv_sample_guid]['isActive'])
-        self.assertDictEqual(response_json['individualsByGuid'], {
-            'I000001_na19675': {'sampleGuids': [sv_sample_guid, existing_index_sample_guid]},
-        })
+        self.assertSetEqual(set(response_json['individualsByGuid']['I000001_na19675']['sampleGuids']),
+                            set([sv_sample_guid, existing_index_sample_guid]))
+
```

Comment:
Eliminated the occasional array element order issue.

Reply:
Can you also assert that the key in `'individualsByGuid'` is `'I000001_na19675'` and the only key in the individual json is `'sampleGuids'`

Reply:
Also `{sv_sample_guid, existing_index_sample_guid}` is cleaner than `set([sv_sample_guid, existing_index_sample_guid])`

Reply:
this is still an issue

Reply:
Fixed now.